docs/accessibility/index.md

```diff
@@ -23,6 +23,9 @@ rewritten:
 Modal.setAppElement(document.getElementById('root'));
 ```
 
+Using a selector that matches multiple elements or passing a list of DOM
+elements will hide all of the elements.
+
```

Comment:
Let's add a note about the risk of using `setAppElement`.

Reply:
Done. Also mentioned the possibility of using a HTMLCollection, as that does update its contents whenever the DOM updates.